stonesoup/metricgenerator/_utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from functools import lru_cache, wraps
+
+
+def clearable_lru_cache():
+    """Cache decorator that allows keeping track of which methods are decorated.
+    A new cache is created for each instance of :class:`SIAPMetrics`
```

Comment:
Probably don't need to reference SIAP here, as could be used for other metrics too.

Reply:
Oops, must be left over from when Id had the cache as a decorator in the SIAP file